worldwind/src/main/java/gov/nasa/worldwind/util/Tile.java

```diff
@@ -399,8 +400,14 @@ protected BoundingBox getExtent(RenderContext rc) {
 
         long elevationTimestamp = rc.globe.getElevationModel().getTimestamp();
         if (elevationTimestamp != this.heightLimitsTimestamp) {
-            Arrays.fill(this.heightLimits, 0);
```

Comment:
Nice catch. As you may have guessed, this mistake results in unnecessarily large bounding volumes for tiles with a min/max that does not include mean sea level.

ElevationModel.getHeightLimits may be a more general place to perform this work, but I like the change here for the time being.